src/components/FoldableText/index.tsx

```diff
@@ -35,7 +37,7 @@ const FoldableText: React.FC<FoldableTextProps> = ({ title, children, ...props }
         </Text>
         <StyledExpandableLabelWrapper>
           <ExpandableLabel expanded={isExpanded} onClick={() => setIsExpanded(!isExpanded)}>
-            {isExpanded ? 'Hide' : 'Details'}
+            {isExpanded ? TranslateString(999, 'Hide') : TranslateString(658, 'Details')}
```

Comment:
Looks like Hide is `1066`

Reply:
I checked the response that returns from crowdin, cannot see 1066 and when i put there it is not translated either. Could it be because our request has limit? 'https://api.crowdin.com/api/v2/projects/422458/languages/de/translations?fileId=8&limit=200'